### PR TITLE
Propose OpenEBS into CNCF Sandbox

### PR DESCRIPTION
**Pull request by kmova**
This PR is to propose OpenEBS into CNCF Sandbox.

Here is a link to the presentation we gave about OpenEBS
to the CNCF Storage WG.
https://docs.google.com/presentation/d/1MEx9EyvFW1cLZhFxUvWVd4bzoRS7PNnMqgmbIZaoulw/edit

OpenEBS Community is pretty excited at the prospect of being in CNCF Sandbox and so are some of the users we reached out to with this idea in the past 2 weeks. All of them saw this is a logical next step for an always open source  project like OpenEBS.

Also we have some traditional storage vendors and cloud service providers as well that will be more able to contribute once OpenEBS is in the CNCF as opposed to being seen as a one vendor project.

Many thanks to the members in the CNCF Storage WG @quinton-hoole, @xing-yang, @chira001 and the TOC members Alexis Richardson and Xiang Li for taking time to listen, coach and help us raise this proposal.

Signed-off-by: kmova <kiran.mova@openebs.io>

### CODE REVIEW

File: proposals/openebs.adoc [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+OpenEBS Source code is split across multiple repositories, some of which are listed below. The complete details of the repositories and their dependencies can be found in: https://github.com/openebs/openebs/blob/master/contribute/design/README.md
+
+* https://github.com/openebs/openebs
+* https://github.com/openebs/maya
+* https://github.com/openebs/jiva
+* https://github.com/openebs/cstor
+* https://github.com/openebs/istgt
+* https://github.com/openebs/node-disk-manager
+
+*Initial Committers and Founding Maintainers:*
```
Comment (caniszczyk) [inline proposals/openebs.adoc]: can you list the companies associated with these maintainers

Comment (kmova) [reply proposals/openebs.adoc]: thanks @caniszczyk! updated.

**Comment by gridworkz** [comment]
We've been using OpenEBS from the start with our project (9 months). Having it part of CNCF would not only boost the relevance of OpenEBS to the community but would help us in promoting our product to our customers. It's a critical part of our infrastructure and I'm sure it will be a critical part of many projects to come. It's just so damn easy to use.

**Comment by ynnt** [comment]
Hey!
We've been using OpenEBS for several months and I can say that it's a great project when you want to store persistent data but have to run a cluster on premises.
OpenEBS is a critical part of our Kubernetes setup, so I'd like to upvote this request as well!

**Comment by blaggacao** [comment]
Slack community has a decent vibe of it's own: Excellent community governance!

File: proposals/openebs.adoc (hunk 1 above)
Comment (xiang90) [reply proposals/openebs.adoc]: also we need to know long the maintainers have been working on the project according to https://github.com/cncf/toc/blob/master/process/project_proposals.adoc

File: proposals/openebs.adoc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+* https://github.com/openebs/istgt
+* https://github.com/openebs/node-disk-manager
+
+*Initial Committers and Founding Maintainers:* 
+
+ * Kiran Mova @kmova (MayaData)
+ * Amit Kumar Das @AmitKumarDas (MayaData)
+ * Karthik Satchitanand @ksatchit (MayaData)
+
+*Additional Maintainers:*
```
Comment (xiang90) [inline proposals/openebs.adoc]: line 81-83 are initial Committers? and line 87-90 + initial committers are current maintainers?

File: proposals/openebs.adoc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+ * Murat Karslioglu @muratkars (MayaData)
+ * Jeffry Molanus @gila (MayaData)
+ * Vishnu Itta @vishnuitta (MayaData)
+ * Richard Elling @richardelling (Viking Enterprise Solutions a division of Sanmina Corporation)
+
+*Infrastructure requests (CI / CNCF Cluster):*
+
+_Development needs:_
+
+none in the short term. We currently use Travis CI, but we may want to use CNCF resources to deploy Jenkins for node E2E test.
```
Comment (xiang90) [inline proposals/openebs.adoc]: Do you pay for Travis CI (for more concurrent job for example) or just the free version for now?

Comment (kmova) [reply proposals/openebs.adoc]: We have setup using the free version for now.

File: proposals/openebs.adoc (hunk 2 above)
Comment (kmova) [reply proposals/openebs.adoc]: That is correct @xiang90

**Comment by sebgoa** [comment]
openebs would be a great addition to CNCF and would foster a neutral ground for future storage developments in the space.

File: proposals/openebs.adoc (hunk 1 above)
Comment (kmova) [reply proposals/openebs.adoc]: updated the start date for each of the maintainers.

**Comment by Pensu** [comment]
I have been playing around with OpenEBS for quite some time. Amazing project with great documentation. The community is very vibrant and helpful. Tight integration with Kubernetes will make it a great addition to CNCF and a big value add to customers.

**Comment by caniszczyk** [comment]
Hey @xiang90 and @monadic, can you confirm that you sponsor this project into the sandbox

**Comment by monadic** [comment]
Confirmed.  I have one question though.  We can discuss on the TOC call

**Comment by xiang90** [comment]
Yes. Confirmed.

**Pull request merged**